extensions/blocks/editor-notes/edit.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress dependencies
+ */
+import { InnerBlocks } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import useEditorNotesMeta from './useEditorNotesMeta';
+import './editor.scss';
+
+export default function EditorNotesEdit( { attributes, className, setAttributes } ) {
+	const { parsedBlocks, setNotes } = useEditorNotesMeta( attributes, setAttributes );
+
+	const onChange = blocks => setNotes( blocks );
+
+	return (
+		<div className={ className }>
+			<p className="wp-block-jetpack-editor-notes__label">{ __( 'Editor Notes', 'jetpack' ) }</p>
+			<InnerBlocks __experimentalBlocks={ parsedBlocks } onChange={ onChange } />
```

Comment:
Just so everyone knows, `__experimentalBlocks` will change to `value` when https://github.com/WordPress/gutenberg/pull/21368 is merged

Reply:
(because it matches the api of `BlockEditorProvider` and basically does the same thing under the hood)

Reply:
When this gets in, let's clean out our internal version: https://github.com/Automattic/wp-calypso/tree/master/apps/o2-blocks/src/editor-notes

Reply:
Noted, thanks!